<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Afficheur</class>
 <widget class="QMainWindow" name="Afficheur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QListWidget" name="listWidget"/>
            </item>
            <item>
             <widget class="QLineEdit" name="inputLine">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="7" column="0" colspan="3">
             <widget class="QPushButton" name="enterButton">
              <property name="text">
               <string>ENTRÃ‰E</string>
              </property>
              <property name="shortcut">
               <string>Return</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="opDivideButton">
              <property name="text">
               <string>/</string>
              </property>
              <property name="shortcut">
               <string>/</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QPushButton" name="deleteButton">
              <property name="text">
               <string>Del</string>
              </property>
              <property name="shortcut">
               <string>Backspace</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="opPlusButton">
              <property name="text">
               <string>+</string>
              </property>
              <property name="shortcut">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="opMinusButton">
              <property name="text">
               <string>-</string>
              </property>
              <property name="shortcut">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="opTimesButton">
              <property name="text">
               <string>*</string>
              </property>
              <property name="shortcut">
               <string>*</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="num7Button">
              <property name="text">
               <string>7</string>
              </property>
              <property name="shortcut">
               <string>7</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="num8Button">
              <property name="text">
               <string>8</string>
              </property>
              <property name="shortcut">
               <string>8</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="num9Button">
              <property name="text">
               <string>9</string>
              </property>
              <property name="shortcut">
               <string>9</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="num4Button">
              <property name="text">
               <string>4</string>
              </property>
              <property name="shortcut">
               <string>4</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="num5Button">
              <property name="text">
               <string>5</string>
              </property>
              <property name="shortcut">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="num6Button">
              <property name="text">
               <string>6</string>
              </property>
              <property name="shortcut">
               <string>6</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="num1Button">
              <property name="text">
               <string>1</string>
              </property>
              <property name="shortcut">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="num2Button">
              <property name="text">
               <string>2</string>
              </property>
              <property name="shortcut">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QPushButton" name="num3Button">
              <property name="text">
               <string>3</string>
              </property>
              <property name="shortcut">
               <string>3</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="num0Button">
              <property name="text">
               <string>0</string>
              </property>
              <property name="shortcut">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="2">
             <widget class="QPushButton" name="spaceButton">
              <property name="text">
               <string>Espace</string>
              </property>
              <property name="shortcut">
               <string>Space</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1" colspan="2">
             <widget class="QPushButton" name="evalButton">
              <property name="text">
               <string>EVAL</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="quoteButton">
              <property name="text">
               <string>'</string>
              </property>
              <property name="shortcut">
               <string>'</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>SIN</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>COS</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>TAN</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>SINH</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>COSH</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>TANH</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>LN</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>LOG</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>SQRT</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="pushButton_10">
              <property name="text">
               <string>SQR</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="pushButton_11">
              <property name="text">
               <string>INV</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="pushButton_12">
              <property name="text">
               <string>CUBE</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QPushButton" name="pushButton_13">
              <property name="text">
               <string>!</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="comboType">
            <property name="toolTip">
             <string>Type de constantes</string>
            </property>
            <item>
             <property name="text">
              <string>Entier</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RÃ©el</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fraction</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboAngles">
            <property name="toolTip">
             <string>Format des angles</string>
            </property>
            <item>
             <property name="text">
              <string>DegrÃ©s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Radians</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxComplexes">
            <property name="text">
             <string>&amp;Complexes</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelStatus">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalculatrice">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouvel_onglet"/>
    <addaction name="actionNouvel_onglet_copie"/>
    <addaction name="actionFermer_onglet"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="separator"/>
    <addaction name="action_apropos"/>
   </widget>
   <widget class="QMenu" name="menu_dition">
    <property name="title">
     <string>Ã‰dition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionRetablir"/>
   </widget>
   <widget class="QMenu" name="menuAfficher">
    <property name="title">
     <string>Afficher</string>
    </property>
    <addaction name="actionPas_de_clavier"/>
    <addaction name="actionClavier_simple"/>
    <addaction name="actionClavier_etendu"/>
   </widget>
   <addaction name="menuCalculatrice"/>
   <addaction name="menu_dition"/>
   <addaction name="menuAfficher"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="action_apropos">
   <property name="text">
    <string>Ã€ propos</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Aide</string>
   </property>
  </action>
  <action name="actionNouvel_onglet">
   <property name="text">
    <string>Nouvel onglet</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionNouvel_onglet_copie">
   <property name="text">
    <string>Nouvel onglet (copie)</string>
   </property>
   <property name="softKeyRole">
    <enum>QAction::NoSoftKey</enum>
   </property>
  </action>
  <action name="actionFermer_onglet">
   <property name="text">
    <string>Fermer l'onglet</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="text">
    <string>Annuler</string>
   </property>
  </action>
  <action name="actionRetablir">
   <property name="text">
    <string>RÃ©tablir</string>
   </property>
  </action>
  <action name="actionPas_de_clavier">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pas de clavier</string>
   </property>
  </action>
  <action name="actionClavier_simple">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clavier simple</string>
   </property>
  </action>
  <action name="actionClavier_etendu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clavier Ã©tendu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
